x-pack/plugin/core/src/main/resources/monitoring-mb-ilm-policy.json

```diff
@@ -0,0 +1,32 @@
+{
+  "phases": {
+    "hot": {
+      "actions": {
+        "rollover": {
+          "max_primary_shard_size": "50gb",
+          "max_age": "3d"
+        }
+      }
+    },
+    "warm": {
+      "actions": {
+        "forcemerge": {
+          "max_num_segments": 1
+        }
+      }
+    },
+    "delete": {
+      "min_age": "${xpack.stack.monitoring.history.duration}",
+      "actions":{
+        "delete": {}
+      }
+    }
+  },
+  "_meta": {
+    "description": "Index lifecycle policy generated for [monitoring-*-8] data streams",
+    "managed": true,
```

Comment:
I'm on the fence for this, I kind of think we should make this unmanaged in that ES will automatically recreate it if it doesn't exist, but it won't touch it if/when a user customizes it. Otherwise we require a user to set up a whole new set up templates just to change the ILM policy.

I think we might want to drop this so that the UI doesn't treat this as a special you-should-not-edit-this-policy item. What do you think?

Reply:
I think this should be a "please edit this policy"  ( that means un-manged right ?)

Reply:
Yeah, so we would drop this line

Reply:
I waffled on this one and left it in since the other policies have it. I'll remove it.